Gems/Atom/Bootstrap/Code/Source/BootstrapSystemComponent.cpp

```diff
@@ -314,13 +315,17 @@ namespace AZ
                 // Create a render pipeline from the specified asset for the window context and add the pipeline to the scene.
                 // When running with no Asset Processor (for example in release), CompileAssetSync will return AssetStatus_Unknown.
                 AzFramework::AssetSystem::AssetStatus status = AzFramework::AssetSystem::AssetStatus_Unknown;
+                default_pipeline_name = BOOTSTRAPSYSTEMCOMPONENT_PIPELINE_NAME;
```

Comment:
This overrides the cvar with a define? Doesn't that defeat the reason to use a cvar?